cartography/data/jobs/cleanup/gcp_iam_domains_cleanup.json

```diff
@@ -0,0 +1,29 @@
+{
+    "statements": [
+        {
+            "query": "MATCH (:GCPCustomer)-[:HAS_DOMAIN]->(n:GCPDomain) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
All the relationships in cleanup files should be linked to GCPProject. Please update this query to show the relationship of these nodes with GCPProject.

Reply:
Also, what does GCPCustomer represent, is it different that GCPUser?

Reply:
Domains are connected to only Customers not to projects. Customer is the entity which has a GCP Account.